tests/unit/lms/views/admin/application_instance_test.py

```diff
@@ -269,6 +269,33 @@ def test_update_instance(
             pyramid_request.route_url("admin.instance.id", id_=application_instance.id)
         )
 
+    @pytest.mark.parametrize(
+        "setting,value,expected",
+        (
+            ("lms_url", "http://some-url.com", "http://some-url.com"),
+            ("custom_canvas_api_domain", "some.domain.com", "some.domain.com"),
+            ("deployment_id", "DEPLOYMENT_ID", "DEPLOYMENT_ID"),
+        ),
+    )
```

Comment:
Could do with a few examples to prove what happens when the values have space or are blank etc.